web_tree_image_tooltip/view/assets.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="assets_backend" name="tree icon assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <link rel='stylesheet' href='/web_tree_image/static/src/scss/common.scss'/>
+            <script type="text/javascript" src="/web_tree_image/static/src/js/tooltip.js"/>
```

Comment:
Can you change the file location according to the new module folder.

Reply:
@tarteo Done